### ISSUE TITLE
use local llama models instead of openai API?

### ISSUE THREAD

**Issue by arjuns2020** [yoheinakajima/prettygraph#8]
can we use local llama models ?

**Comment by mNandhu** [comment]
Yes, You need to change ~line 55 in main.py to, result = generate_text_completion("ollama/llama3.1").
Check LiteLLM [Docs](https://docs.litellm.ai/docs/providers/ollama) for more info.
